dcos/auth.py

```diff
@@ -58,10 +57,16 @@ def _prompt_user_for_token(url, token_type):
     :rtype: str
     """
 
-    msg = "\n{}\n\n    {}\n\nEnter {}:".format(
-          "Please go to the following link in your browser:",
+    msg = "\n{}\n\n{}\n\n    {}\n\nEnter {}:".format(
+          "Attempting to automatically open a page in your browser...",
+          "If it doesn't work, open the following link:",
```

Comment:
Take a look at 9ae2e47ac944237b3512fcaaa7f5a5f12be02496. I used your words and also took the liberty of reworking how the message is constructed, because I found the old way a bit hard to read.